### PR TITLE
Create bg-bg/perl.html.markdown

### PR DESCRIPTION
**Pull request by kberov**
- [  ] PR touches only one file (or a set of logically related files with similar changes made)
- [  ] Content changes are aimed at *intermediate to experienced programmers* (this is a poor format for explaining fundamental programming concepts)
- [  ] YAML Frontmatter formatted according to [CONTRIBUTING.md](https://github.com/adambard/learnxinyminutes-docs/blob/master/CONTRIBUTING.markdown)
  - [  ] Seriously, look at it now. Watch for quotes and double-check field names.

### CODE REVIEW

File: bg-bg/perl.html.markdown [hunk 1]
```diff
@@ -0,0 +1,326 @@
+---
+name: perl
+category: language
+language: perl
+filename: learnperl.pl
```
Comment (vendethiel) [inline bg-bg/perl.html.markdown]: please add `-bg` suffix to the filename :)

**Comment by kberov** [comment]
ok, thanks, doing it

**Comment by kberov** [comment]
done

**Comment by vendethiel** [comment]
Thanks! :star:

**Pull request merged**